File: Java Projects/Coffee Machine (Easy)/Stage 3/CoffeeMachine.java
package machine;

import java.util.Scanner;

public class CoffeeMachine {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Write how many ml of water the coffee machine has:");
        int numPossibleCups = scanner.nextInt() / 200;
        System.out.println("Write how many ml of milk the coffee machine has:");
        int mlMilk = scanner.nextInt();
        numPossibleCups = (mlMilk / 50) < numPossibleCups ? (mlMilk / 50) : numPossibleCups;
        System.out.println("Write how many grams of coffee beans the coffee machine has:");
        int gramsBeans = scanner.nextInt();
        numPossibleCups = (gramsBeans / 15) < numPossibleCups ? (gramsBeans / 15) : numPossibleCups;
        System.out.println("Write how many cups of coffee you will need:");
        int numCups = scanner.nextInt();
        if (numPossibleCups == numCups) {
            System.out.println("Yes, I can make that amount of coffee");
        } else if (numPossibleCups > numCups) {
            System.out.println("Yes, I can make that amount of coffee (and even " + (numPossibleCups - numCups) + " more than that)");
        } else {
            System.out.println("No, I can only make " + numPossibleCups + " cup(s) of coffee");
        }
        
    }
}

File: Python Projects/Coffee Machine (Easy)/Stage 2/coffee_machine.py
ML_WATER_PER_CUP_COFFEE = 200
ML_MILK_PER_CUP_COFFEE = 50
G_BEANS_PER_CUP_COFFEE = 15

print("Write how many cups of coffee you will need: ")
num_cups = int(input())
print(f"For {num_cups} cups of coffee you will need: ")
print(f"{num_cups * ML_WATER_PER_CUP_COFFEE} ml of water")
print(f"{num_cups * ML_MILK_PER_CUP_COFFEE} ml of milk")
print(f"{num_cups * G_BEANS_PER_CUP_COFFEE} g of coffee beans")
